### ISSUE TITLE
Add default forum permissions setting

### ISSUE THREAD

**Issue by joshuamonroe** [uchicago-cs/chigame#412]
During testing I noticed that regular user accounts do not have permissions to view the forums.

This is the forum viewed as an admin. Thus far, most testing has been done from an admin account.
![Image](https://github.com/uchicago-cs/chigame/assets/59244307/00c2b79b-cdc0-423e-9bdd-87d81d157d56)

And this is the same forum viewed as a regular user.
![Image](https://github.com/uchicago-cs/chigame/assets/59244307/839d2b0c-a7d5-4dd7-985e-3365998ba6ef)

A fix would be pretty easy, we would just need to set django-machina's default authenticated user permissions setting according to the defaults provided [here](https://django-machina.readthedocs.io/en/latest/settings.html#machina-default-authenticated-user-forum-permissions). This would apply to all authenticated users across all forums that don't have more restrictive permissions.
```
MACHINA_DEFAULT_AUTHENTICATED_USER_FORUM_PERMISSIONS = [
    'can_see_forum',
    'can_read_forum',
    'can_start_new_topics',
    'can_reply_to_topics',
    'can_edit_own_posts',
    'can_post_without_approval',
    'can_create_polls',
    'can_vote_in_polls',
    'can_download_file',
]
```
Alternatively, we could consider setting permissions manually on a per forum basis, more info on what that would entail and a full list of possible permissions can also be found [here](https://django-machina.readthedocs.io/en/latest/forum_permissions.html).

**Comment by joshuamonroe** [comment]
I've added the the necessary permissions setting and submitted a pull request to merge it into dev
- #414

**Comment by joshuamonroe** [comment]
Closed after successfully merging and closing #414. Thanks to @bburgess19 for reviewing the PR.

**Comment by majorsylvie** [comment]
Issue Score: Excellent
